htdocs/multicurrency/multicurrency_rate.php

```diff
@@ -32,6 +32,10 @@
  *  \brief      Page to list multicurrency rate
  */
 
+if (!defined('NOTOKENRENEWAL')) {
```

Comment:
There is no reason to add this. This introduce a security hole.
Only php files that are "ajax called" files should have NOTOKENRENEWAL set

Reply:
ok , i added a token to the second form and remove NOTOKENRENEWAL. 
thx for your time.

Reply:
i misundestrood your last  comment. 
i realise you added the token ... sorry